LearningLens2025/frontend/lib/Views/view_submissions.dart

```diff
@@ -494,6 +521,38 @@ class SubmissionListState extends State<SubmissionList> {
                                                           .toList(),
                                                     ),
                                                     SizedBox(height: 4),
+
+                                                    // Grade Level Dropdown
+                                                    DropdownButtonFormField<
+                                                        String>(
+                                                      value:
+                                                          gradeLevelSelectionMap[
+                                                                  participant
+                                                                      .id] ??
+                                                              'College',
+                                                      decoration:
+                                                          InputDecoration(
+                                                              labelText:
+                                                                  'Grade Level'),
+                                                      onChanged: (newValue) =>
+                                                          _handleGradeLevelChanged(
+                                                              participant.id,
+                                                              newValue),
+                                                      items: [
+                                                        'Elementary School',
+                                                        'Middle School',
+                                                        'High School',
+                                                        'College'
+                                                      ]
```

Comment:
There's a constant of these, please use it for consistency with other views - examples are in the analytics and rubric creation views.

Reply:
Looking at those learninglensconstants it's going to be more problematic longer term referencing that I think since it isn't a map, its a list. I would have to hard code the position. Also, having the available options listed in the actual view submissions is consistent with the other option choices as well.

Reply:
I think you misunderstood what I meant here.
```suggestion
                                                      items: LearningLensConstants.gradeLevels
```

Reply:
See analytics page as an example:
<img width="606" height="396" alt="image" src="https://github.com/user-attachments/assets/eef570e4-d2be-415c-85a2-1bed993f97ba" />